solidity/test/bridge/Bridge.Governance.test.ts

```diff
@@ -0,0 +1,4339 @@
+          .timestamp
+        const initTimestamp = (await tx.wait()).events[0].args.timestamp
+        const elapsedTime = BigNumber.from(blockTimestamp).sub(initTimestamp)
+
+        expect(
+          BigNumber.from(constants.governanceDelay).sub(elapsedTime)
+        ).to.be.equal(constants.governanceDelay)
+      })
+
+      it("should emit DepositDustThresholdUpdateStarted event", async () => {
+        const blockTimestamp = (await ethers.provider.getBlock(tx.blockNumber))
+          .timestamp
+        await expect(tx)
+          .to.emit(bridgeGovernance, "DepositDustThresholdUpdateStarted")
+          .withArgs(1337, blockTimestamp)
+      })
+    })
+  })
+
+  describe("finalizeDepositDustThresholdUpdate", () => {
+    context("when the caller is not the owner", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(thirdParty)
+            .finalizeDepositDustThresholdUpdate()
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when the update process is not initialized", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(governance)
+            .finalizeDepositDustThresholdUpdate()
+        ).to.be.revertedWith("Deposit dust threshold must be greater than zero")
+      })
+    })
+
+    context("when the deposit dust threshold has not passed", () => {
+      before(async () => {
+        await createSnapshot()
+
+        await bridgeGovernance
+          .connect(governance)
+          .beginDepositDustThresholdUpdate(7331)
+
+        await helpers.time.increaseTime(constants.governanceDelay - 60) // -1min
+      })
+
+      after(async () => {
+        await restoreSnapshot()
+      })
+
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(governance)
+            .finalizeDepositDustThresholdUpdate()
+        ).to.be.revertedWith("Governance delay has not elapsed")
+      })
+    })
+
+    context(
+      "when the update process is initialized and governance delay passed",
+      () => {
+        let tx: ContractTransaction
+
+        before(async () => {
+          await createSnapshot()
+
+          await bridgeGovernance
+            .connect(governance)
+            .beginDepositDustThresholdUpdate(7331)
+
+          await helpers.time.increaseTime(constants.governanceDelay)
+
+          tx = await bridgeGovernance
+            .connect(governance)
+            .finalizeDepositDustThresholdUpdate()
+        })
+
+        after(async () => {
+          await restoreSnapshot()
+        })
+
+        it("should update the deposit dust threshold", async () => {
+          const { depositDustThreshold } = await bridge.depositParameters()
+          expect(depositDustThreshold).to.be.equal(7331)
+        })
+
+        it("should emitDepositDustThresholdUpdated event", async () => {
```

Comment:
should emitDepositDustThresholdUpdated event ->
should emit DepositDustThresholdUpdated event